notebooks/official/tabular_workflows/prophet_on_vertex_pipelines.ipynb

```diff
@@ -726,7 +642,7 @@
         "id": "ySuRn2i0rjLH"
       },
       "source": [
-        "You can take a look at the sales data that was generated. Later in this tutorial, we visualize the time series along with our forecast.\n",
+        "You can take a look at the sales data that was generated. Later in this tutorial, visualize the time series along with our forecast.\n",
```

Comment:
remove "our" and use "the". Suggested edit:

- Later in this tutorial, the time series is visualized along with the forecast.